lib/ssl/src/tls_sender.erl

```diff
@@ -488,14 +488,12 @@ send_application_data(Data, From, StateName,
 	    {Msgs, ConnStates} = tls_record:encode_data(Data, Version, ConnStates0),
 	    case tls_socket:send(Transport, Socket, Msgs) of
                 ok when From =:= dist_data ->
-                    ssl_logger:debug(get(log_level), outbound, 'record', Msgs),
```

Comment:
I think this should be ok :) Although it is a change in behavior as this affects the log_level option, I think it will mostly be used to debug the handshake  and seeing user data packets is not that interesting. I am not sure what OpenSSL does in their similar debug mode, maybe we should check!